api/data.go

```diff
@@ -67,3 +67,24 @@ func (api *API) AddDataSets(ctx context.Context) http.HandlerFunc {
 		}
 	}
 }
+
+// GetCacheTime retrieves cache time data for a given ID and writes it to the HTTP response.
+func (api *API) GetCacheTime(ctx context.Context, w http.ResponseWriter, req *http.Request) {
+	log.Info(ctx, "calling get cache time handler")
+
+	vars := mux.Vars(req)
+	id := vars["id"]
+
+	cacheTime, err := api.dataStore.GetCacheTime(ctx, id)
```

Comment:
Don't think you're catching what happens if they don't supply an `id` here - i.e. a 400 response.

Reply:
This will be covered as part of another ticket.